### DISCUSSION TITLE
Add easy way to use frontend dev servers inside photino

### DISCUSSION THREAD

**Discussion by manuel3108** [tryphotino/photino.NET#121] [Ideas]
First of all: I love this project. I wish I would have found out earlier about this!

After looking through some of the examples you provided, I asked myself: How great would it be, if every start of the app would also spin up a dev server of my preferred frontend framework (svelte-(kit) in my case) and directly show it inside the app?

This sounded so interesting to me, that I needed to investigate....
So here I am: And I can say, that it's definitely possible!

Here is a small video demo, I just made: 

https://user-images.githubusercontent.com/30698007/218324386-1c07063e-133a-4183-824b-3adae789d126.mp4

Here is a small POC repo in case you want to try it out yourself:
https://github.com/manuel3108/photino-sveltekit-dev-server

I think it would be a great addition to photino & especially the samples to add such a feature, without adding a bunch of code likeit was necessary here.

What's your opinion?

**Comment by ottodobretsberger** [comment]
We like it! We would like to integrate this as an optional dev experience for people, but we would like to make it a little bit more generalized and integrate it with our server package. We will spend some time looking into this to get it added for one of our next releases, and we might reach out to your for additional input.

**Reply by manuel3108** [reply]
Sounds great.
Generalizing sounds great, as this was only a quick POC

**Comment by rtmann** [comment]
Hmm, I think I now want to tinker with making a VS Code extension to host Photino's debug tools in VS Code and get it working with server side client debugging like MS Edge Dev Tools.

Would probably only work on windows though... Microsoft has not made a WebView2 for Linux.... It's like they don't care....
